packaging/osx/scripts/postinstall

```diff
@@ -4,6 +4,9 @@
 # Licensed under the MIT license. See LICENSE file in the project root for full license information.
 #
 
+# A temporary fix for the permissions issue(s)
+chmod -R 755 $2
```

Comment:
@anurse that is a pretty good idea, actually. Though, for reference, the $1 to $n are pretty well defined for pkg creation on OS X.